Towny/src/main/java/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -2256,6 +2256,10 @@ public static EnumSet<Material> getPlotManagementIgnoreIds() {
 		return toMaterialEnumSet(getStrArr(ConfigNodes.NWS_PLOT_MANAGEMENT_REVERT_IGNORE));
 	}
 
+	public static EnumSet<Material> getRevertOnUnclaimWhitelistMaterials() {
```

Comment:
This return type should be a Collection for future proofing with upcoming spigot changes.